### ISSUE TITLE
Accessibility Issue: Violation of WCAG 2.0 AA Colour Contrast Ratios /exercises-automation-npm.html

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#723]
**Description**
Relates to #714 ... Several violations in colour contrast issue

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/exercises-automation-npm.html
*CSS Selector(s) (if possible and applicable)*:  
```p:nth-child(12) > .language-plaintext```
```p:nth-child(22) > .language-plaintext:nth-child(1)```
```p:nth-child(22) > .language-plaintext:nth-child(2)```
```p:nth-child(26) > .language-plaintext```
```p:nth-child(28) > .language-plaintext:nth-child(1)```
```p:nth-child(28) > .language-plaintext:nth-child(2)```
```.language-plaintext:nth-child(3)```
```p:nth-child(31) > .language-plaintext```
```p:nth-child(46) > .language-plaintext```
```p:nth-child(48) > .language-plaintext```
```p:nth-child(57) > .language-plaintext```
```p:nth-child(60) > .language-plaintext```
```.nb:nth-child(34)```
```.nb:nth-child(35)```

*Additional Identifier(s) (if possible and applicable)*:  
*Additional Details*:  

**Screenshot(s) (if possible)**

**Additional information or thoughts**

As a result of scan #668
